devpi/entrypoint.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+# Adapted from: https://github.com/LordGaav/docker-devpi
+
+function generate_password() {
+    # We disable exit on error because we close the pipe
+    # when we have enough characters, which results in a
+    # non-zero exit status
+    set +e
+    tr -cd '[:alnum:]' < /dev/urandom | fold -w30 | head -n1 | tr -cd '[:alnum:]'
+    set -e
+}
```

Comment:
Maybe we could set some fixed password to make it simpler? Of in a CI env variable if you prefer to keep it a secret. We don't really care about it being random for testing.